ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/TransactionServiceIT.scala

```diff
@@ -65,19 +68,29 @@ class TransactionServiceIT
     with Inside
     with AsyncTimeLimitedTests
     with TestExecutionSequencerFactory
-    with TransactionServiceHelpers
     with ParameterShowcaseTesting
     with OptionValues
-    with Matchers
-    with TestTemplateIds {
+    with Matchers {
 
-  override protected val config: Config =
+  override protected def config: Config =
     Config.default.withTimeProvider(TimeProviderType.WallClock)
 
+  protected val helpers = new TransactionServiceHelpers(config)
+  protected val testTemplateIds = new TestTemplateIds(config)
+  protected val templateIds = testTemplateIds.templateIds
+
+  val runSuffix = "-" + Random.alphanumeric.take(10).mkString
```

Comment:
these utilities code go into a test helper library as we will need them in other places too I suppose.